metaflow/plugins/kfp/Dockerfile

```diff
@@ -62,3 +62,4 @@ RUN echo "%users ALL=(ALL) NOPASSWD:/usr/local/bin/pip" >> /etc/sudoers && \
     echo "%users ALL=(ALL) NOPASSWD:/usr/local/bin/python" >> /etc/sudoers && \
     echo "%users ALL=(ALL) NOPASSWD:/usr/bin/apt-get" >> /etc/sudoers && \
     echo "%users ALL=(ALL) NOPASSWD:/usr/bin/apt" >> /etc/sudoers
+
```

Comment:
typo?

Reply:
Can revert, but this file is missing end of file new line.

Reply:
I assume this new line is fine to be added. Marking as resolved.